client/subcomponents/produce/ProduceForm/ProduceForm.jsx

```diff
@@ -64,30 +69,44 @@ export default function ProduceForm({
                 </Field>
               </div>
 
-              <ul>
-                {gardens?.length ? (
-                  gardens.map((garden) => {
-                    return (
-                      <li key={garden.id} value="hello">
-                        <Field
-                          value={garden.id.toString()}
-                          type="checkbox"
-                          name="gardenIds"
-                        />
-                        {errors.gardenIds && touched.gardenIds ? (
-                          <p>{errors.gardenIds}</p>
-                        ) : null}
-                        {garden.name}
-                      </li>
-                    )
-                  })
-                ) : (
-                  <p>No produce yet</p>
-                )}
-              </ul>
+              <div className="pt-0 flex flex-col space-y-4 tracking-wide">
+                <h1 className="font-semibold font-sans text-xl mt-10 ">
+                  Gardens
+                </h1>
+                <ul>
+                  {gardens?.length ? (
+                    gardens.map((garden) => {
+                      return (
+                        <li
+                          key={garden.id}
+                          value="hello"
+                          className="leading-10"
+                        >
+                          <Field
+                            value={garden.id.toString()}
+                            input
+                            type="checkbox"
+                            class="checked:accent-orange text-white mr-5"
+                            name="gardenIds"
+                          />
+                          {errors.gardenIds && touched.gardenIds ? (
+                            <p>{errors.gardenIds}</p>
+                          ) : null}
+                          {garden.name}
+                        </li>
+                      )
+                    })
+                  ) : (
+                    <p>No produce yet</p>
+                  )}
+                </ul>
+              </div>
 
-              <div className="button-group">
-                <button className="submit form-box" type="submit">
+              <div className="rounded-md pt-14 pl-40">
+                <button
+                  type="button"
+                  className="inline-block px-10 py-2 bg-orange text-white font-sans font-bold text-xs leading-tight  rounded shadow-md backdrop:focus:outline-none focus:ring-0 active:bg-orange active:shadow-lg transition duration-150 ease-in-out"
```

Comment:
That's a cool button.

I can imagine this sort of styling being used on many buttons in the project.  In which case, it makes sense to move the styling to a custom tailwind class:

**client/css/main.css**
`@layer components {
  .submit-button {
    @apply inline-block px-10 py-2 bg-orange text-white font-sans font-bold text-xs leading-tight  rounded shadow-md backdrop:focus:outline-none focus:ring-0 active:bg-orange active:shadow-lg transition duration-150 ease-in-out
  }
}`

And apply that custom style to your button like this:

**client/subcomponents/produce/ProduceForm/ProduceForm.jsx**
`<button type="button" className="submit-button">`